ls1intum/Apollon
# Bugfix/88 minor ui issues

### Checklist

### Motivation and Context
Fix for #88 + We want more users to report their issues with Apollon so that we can improve

### Description
Changes as requested in issue
Does not work for Firefox, just for chrome so far. We will change that when we include a proper framework for the standalone version (right now vanilla javascript + html + css)

### Steps for Testing

1. select diagram type which is too long for dropdown and check if dropdown adapts
2. select diagram type "Component Diagram" and check if interface preview is correct

Component-Diagram Interface Preview
### Screenshots
Dropdown + read-only field
![minor_bugs_interface](https://user-images.githubusercontent.com/22955066/77072782-8a33b980-69ee-11ea-8bb2-74c1675e7be2.png)

Dropdown + read-only field
![ui_minor_bugs_dropdown_read_only](https://user-images.githubusercontent.com/22955066/77072792-8d2eaa00-69ee-11ea-95e3-21eb88ece44a.png)
